docs/src/main/paradox/scala/http/server-side/http2.md

```diff
@@ -0,0 +1,146 @@
+# Server-Side HTTP/2 (Preview)
+
+Server-Side HTTP/2 support in akka-http is currently available as a preview.
+This means it is ready to be evaluated, but the API's and behavior is likely to change.
+
+## Enable HTTP/2 support
+
+`akka-http2-support` must be added as a dependency:
+
+sbt
+:   @@@vars
+    ```sbt
+    "com.typesafe.akka" %% "akka-http2-support" % "$project.version$"
+    ```
+    @@@
+
+maven
+:   @@@vars
+    ```
+    <dependency>
+        <groupId>com.typesafe.akka</groupId>
+        <artifactId>akka-http2-support_2.12</artifactId>
+        <version>$project.version$</version>
+    </dependency>
+    ```
+    @@@
+
+HTTP/2 can then be enabled through configuration:
+
+```
+akka.http.server.preview.enable-http2 = on
+```
+
+## Use `bindAndHandleAsync` and HTTPS
+
+We only support HTTP/2 over HTTPS: browsers don't typically support HTTP/2 on plain HTTP, and other clients that do are rare. See the @ref[HTTPS section](../server-side-https-support.md) for how to set up HTTPS.
+
+You can use @scala[@scaladoc[Http().bindAndHandleAsync](akka.http.scaladsl.HttpExt)]@java[@javadoc[new Http().bindAndHandleAsync()](akka.http.javadsl.HttpExt)] as long as you followed the above steps:
```

Comment:
pretty sure that's not how it's used :-) `new Http()` 
Should be `Http.create(system)` I believe :)